### ISSUE TITLE
Overly large hover area for build graph

### ISSUE THREAD

**Issue by ncameronbritt** [openshift/origin-web-console#2839]
Hovering on the graph shows a tooltip with info, but the hover area is much bigger than the bar. This makes for some unexpected behavior when moving the mouse. Notice mouse and tooltip positions:
![screen shot 2018-02-21 at 1 43 09 pm](https://user-images.githubusercontent.com/8974593/36498813-7d8063ea-170d-11e8-9aa1-4329644d0c35.png)

**Comment by TheRealJon** [comment]
I looked into this and it seems that this is the expected behavior for c3 charts.

https://codepen.io/TheRealJon/pen/oEMOdW

I looked through the C3 docs and there does not seem to be an easy way to customize or change this behavior. If we think it's worth while, we might be able to dig a little deeper and maybe contribute a new feature to the C3 repo to enable customization of the tooltip hover behavior.
